components/package.json

```diff
@@ -1,19 +1,20 @@
 {
     "name": "@brightlayer-ui/react-components",
-    "version": "6.0.0",
+    "version": "6.1.0",
     "description": "React components for Brightlayer UI applications",
     "scripts": {
         "test": "jest src",
         "build": "tsc --p tsconfig.lib.json",
         "watch:storybook": "tsc --p tsconfig.lib.json --watch --outdir ../demos/storybook/node_modules/@brightlayer-ui/react-components",
-        "lint": "eslint \"src/**/**.{tsx,ts}\"",
+        "lint": "eslint \"src/**.{tsx,ts}\"",
         "lint:fix": "eslint \"src/**/**.{tsx,ts}\" --fix",
         "prettier": "prettier \"src/**/**.{ts,tsx,js,jsx,json,css,scss,html}\" --write",
         "prettier:check": "prettier \"src/**/**.{ts,tsx,js,jsx,json,css,scss,html}\" --check"
     },
     "prettier": "@brightlayer-ui/prettier-config",
     "dependencies": {
-        "@brightlayer-ui/colors": "^3.0.0",
+        "@brightlayer-ui/colors": "3.0.1",
+        "@emotion/css": "^11.9.0",
```

Comment:
Now that I'm doing a little more reading, why are we switching to CX over CLSX? 

Comment on an issue from one of the MUI developers:
<img width="919" alt="image" src="https://user-images.githubusercontent.com/29152776/174628948-709053d4-5c48-4aa1-acb3-4a60d62a6fee.png">

Is our class merging logic still working as expected where we have replaced clsx with cx? It seems like it might be doing more under the hood than we were expecting.

Reply:
MUI doesn't use CX anywhere in their code (but they use CLSX a lot)

Reply:
Colors & Colors Branding just need to be demo'd to Joe before they can get published; otherwise they are in a good spot.

Reply:
Looks like that CX comment is within the context of using TSS.

I'm not opposed to using it, if it works...We probably should take 10 minutes and test out a few things to make sure the classes getting passed in via className, classes, and the default classes are getting merged together in the correct way (I think they are).

I'm also not opposed to continuing to use clsx since we don't have to add a new dependency and that's how MUI is continuing to do it today.